### PR TITLE
Feature/displaying leaderboard data

### PR DESCRIPTION
**Pull request by itsninam**

### CODE REVIEW

File: src/Components/Leaderboard.js [hunk 1]
```diff
@@ -2,72 +2,92 @@
...
+        for (let key in dataResponse) {
+          leaderboardEntries.push({
+            key: key,
+            score: dataResponse[key].score,
+            username: dataResponse[key].username,
+          });
+        }
+        setDbData(sortScore(leaderboardEntries));
+      } else {
+        console.log("Can't retrieve data");
```
Comment (alextfriedman) [inline src/Components/Leaderboard.js:52]: are we going to change this to an error message that shows up on screen instead of a console log?

**Review by alextfriedman — CHANGES_REQUESTED** [review]
Great work! 💅

**Review by benbourgon — APPROVED** [review]

**Review by mwazir — APPROVED** [review]
Looks good!

**Comment by itsninam** [comment]
merging after PR review

**Pull request merged**